pkgs/tools/misc/xdiskusage/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchurl, fltk }:
+
+stdenv.mkDerivation rec {
+  pname = "xdiskusage";
+  version = "1.60";
+
+  src = fetchurl {
+    urls = [
+      "https://xdiskusage.sourceforge.net/${pname}-${version}.tgz"
+    ];
+    hash = "sha256-e1NtxvG9xtm+x8KblDXCPZ0yv/ig6+4muZZrJz3J9n4=";
+  };
+
+  nativeBuildInputs = [ fltk ];
```

Comment:
Afaicr, it should be a `buildInput`.

Reply:
I think (but may be wrong) it should be nativeBuildInput as it provides the bin/fluid that is used at build time.